lib/cosmos/tools/tlm_grapher/data_objects/housekeeping_data_object.rb

```diff
@@ -252,14 +252,10 @@ def process_packet(packet, count)
 
         # Prune Data
         prune_to_max_points_saved()
-      rescue Exception => error
```

Comment:
I removed this blanked rescue because it was masking the original bug. Might be dangerous but we need to know if there are issues when we run the AHK tests.